src/actions.ts

```diff
@@ -116,16 +130,9 @@ export async function executeActions(
           },
           "please ensure this environmental variable is not set because graphile-migrate sets it dynamically for children.",
         ),
-        encoding: "utf8",
-        // 50MB of log data should be enough for any reasonable migration... right?
-        maxBuffer: 50 * 1024 * 1024,
+        shell: true,
```

Comment:
So we're using the `shell: true` option to make it so commands such as `graphile-worker --schema-only` still work even though we're not passing args. Interesting.